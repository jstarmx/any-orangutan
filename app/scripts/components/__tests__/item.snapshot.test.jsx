import React from 'react';
import renderer from 'react-test-renderer';

import Item from '../item';

const props = {
  date: '2017-06-03T14:44:43-08:00',
  link: 'http://www.flickr.com/photos/thevixen/34702706110/',
  path: 'http://farm5.staticflickr.com/4266/34702706110_e722ea7bf8_m.jpg',
  title: 'Make Them Go Away',
};

it('renders correctly', () => {
  const tree = renderer.create(
    <Item { ...props } />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
