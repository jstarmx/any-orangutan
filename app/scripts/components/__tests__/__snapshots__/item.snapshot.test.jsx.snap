// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="item"
>
  <a
    href="http://www.flickr.com/photos/thevixen/34702706110/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <img
      alt="Make Them Go Away"
      className="item__image"
      src="http://farm5.staticflickr.com/4266/34702706110_e722ea7bf8_m.jpg"
    />
  </a>
  <p
    className="item__title"
  >
    Make Them Go Away
  </p>
  <p
    className="item__date"
  >
    3rd Jun 2017
  </p>
  <button
    className="item__button"
  >
    <img
      alt="Not currently favourited"
      className="item__icon"
      src="heart_empty.svg"
    />
    Favourite
  </button>
</div>
`;
